vyper/codegen/core.py

```diff
@@ -304,7 +304,8 @@ def copy_bytes(dst, src, length, length_bound):
 def get_bytearray_length(arg):
     typ = UINT256_T
 
-    # TODO add "~empty" case to mirror get_dyn_array_count
+    if arg.value == "~empty":
```

Comment:
please leave a comment saying that it would be nice to merge the implementations of `get_bytearray_length` and `get_dyn_array_count`